app/pages/adduser/errorStrings.js

```diff
@@ -0,0 +1,11 @@
+export const errorMessages = {
+  EmailRequired: 'Email address is required',
+  EmailTooLong: 'Email address is too long',
+  EmailInvalidFormat: 'Email address format is not valid',
+  EmailAlreadyExists: 'Email address already exists',
+  PhoneNumberRequired: 'Telephone number is required',
+  FirstNameRequired: 'First name is required',
+  FirstNameTooLong: 'First name is too long',
+  LastNameRequired: 'Last name is required',
+  LastNameTooLong: 'Last name is too long',
```

Comment:
I am just wondering why you didn't put these in the manifest as with MP? Does the new API response make this more tricky to do it that way

Reply:
yeah good shout - I will move them 👍